Mobile Test Runner for Jama/Mobile Test Runner for Jama.xcodeproj/project.pbxproj

```diff
@@ -627,7 +631,7 @@
 				DEVELOPMENT_TEAM = 5BL5M2XBM2;
 				INFOPLIST_FILE = "Mobile Test Runner for Jama/Info.plist";
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = Jama;
+				PRODUCT_BUNDLE_IDENTIFIER = Jaca;
```

Comment:
Thanks for changing this, the Jama identifier was causing problems when I tested on my phone